### ISSUE TITLE
Inclusion in JDK

### ISSUE THREAD

**Issue by krisskross** [JakeWharton/byteunits#5]
This be a great addition to the JDK. Why not propose it as a JEP?

**Comment by JakeWharton** [comment]
Yeah. I'm definitely not against it. I really don't know the process, though. I'll have to look more into it.

**Comment by krisskross** [comment]
I don't have any experience with JEPs to share but hopefully it won't be too painful. Maybe it can be included existing JEPs?
